src/lib/Preview.js

```diff
@@ -109,7 +109,7 @@ class Preview extends EventEmitter {
     /** @property {Object} - Map of disabled viewer names */
     disabledViewers = {};
 
-    /** @property {string} - Access token */
+    /** @property {string|Function} - Access token */
```

Comment:
Hmm... do we actually store the token on the Preview instance? If not, maybe we can remove this